### ISSUE TITLE
Change icons in results.html from absolute path to relative path

### ISSUE THREAD

**Issue by PaulYuuu** [avocado-framework/avocado#3240]
https://github.com/avocado-framework/avocado/blob/ec3b16c413344cf88a70c0d2e154a220157de13c/optional_plugins/html/avocado_result_html/templates/results.html#L86-L87

Sometimes, I will copy job results to a remote server for analysis, but `logs_icon.svg` and `whiteboard_icon.svg` both point to absolute path.
Could we copy the `images` directory to the result_dir and remove`data.source_path`?

### Actual
```html
            <div>
              <a class="col-xs-1" href="test-results/001-Host_RHEL.m8.u0.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.0.0.ppc64le.io-github-autotest-qemu.pci_bridge.basic.one_bridge/debug.log"><img src="/home/kar/workspace/avocado/optional_plugins/html/avocado_result_html/templates/images/logs_icon.svg" title="Debug log"/></a>
              <div class="col-xs-1 hiddenText"><img src="/home/kar/workspace/avocado/optional_plugins/html/avocado_result_html/templates/images/whiteboard_icon.svg"/></div>
            </div>
```
### Expected
```html
            <div>
              <a class="col-xs-1" href="test-results/001-Host_RHEL.m8.u0.qcow2.virtio_blk.up.virtio_net.Guest.RHEL.7.7.ppc64.io-github-autotest-qemu.unattended_install.cdrom.extra_cdrom_ks.default_install.aio_threads/debug.log"><img src="images/logs_icon.svg" title="Debug log"/></a>
              <div class="col-xs-1 hiddenText"><img src="images/whiteboard_icon.svg"/></div>
            </div>

**Comment by PaulYuuu** [comment]
Another problem:
https://github.com/avocado-framework/avocado/blob/ec3b16c413344cf88a70c0d2e154a220157de13c/optional_plugins/html/avocado_result_html/templates/style.css#L9
If a line in the whiteboard is too long, `white-space: pre;` will only breakon `\n` and `<br>`, maybe uses `white-space: pre-wrap;` be better.
`white-space: pre;`: Lines break only on new lines and \<br\>
`white-space: pre-wrap;`: Lines break on new lines, \<br\>, but also when reaching the end of the element

**Comment by richtja** [comment]
Hi @PaulYuuu, you are right that it is a problem when images are not loaded, but I think it's better to use `alt` tag or something like that than copy images to the result_dir.

Good catch with `white-space: pre;`, the `white-space: pre-wrap;` is much better.

If you have the capacity to fix these things the PR is welcome or I will look at that around next week.

**Comment by clebergnu** [comment]
Hi @PaulYuuu , on the topic of the images, would it work if we just add a `alt` field to those? I fear that copying the images over to every single result is going to clutter the user's filesystem.

**Comment by PaulYuuu** [comment]
Hi @richtja @clebergnu , The `alt` field is limited by width, so it does not work well.
> but I think it's better to use `alt` field or something like that than copy images to the result_dir.

> I fear that copying the images over to every single result is going to clutter the user's filesystem.

Agree with you, copy to every single result_dir is a bad workaround.
Have an idea: inline svg in html/css, it might be better than loading svg files from outside.
I'll try it once, thanks all.

**Comment by richtja** [comment]
fixed in #3246
